projects/common/src/telemetry/track/track.directive.ts

```diff
@@ -58,8 +58,12 @@ export class TrackDirective implements OnInit, OnChanges, OnDestroy {
   }
 
   private trackUserEvent(userEvent: string, eventObj: MouseEvent): void {
+    const targetElement = eventObj.target as HTMLElement;
     this.userTelemetryImplService.trackEvent(`${userEvent}: ${this.trackedEventLabel}`, {
-      ...(eventObj.target as HTMLElement),
+      tagName: targetElement.tagName,
+      innerHTML: targetElement.innerHTML,
```

Comment:
This still feels like over-reporting (and potentially can report PII data). Why do we need the innerHTML of a clicked (for example) element? Can we start minimal and add stuff as use cases come up?

Reply:
I can remove innerHtml. But btw, you need to put controls in your telemetry tool to protect against storing pii. It shouldn't be completely dependent on the code

Reply:
done